wiki/release/release-process.md

```diff
@@ -303,17 +287,41 @@ https://github.com/apache/pulsar/releases/tag/v2.X.0-candidate-1
 Pulsar's KEYS file containing PGP keys we use to sign the release:
 https://dist.apache.org/repos/dist/dev/pulsar/KEYS
 
+Docker images:
+
+<link of the pulsar images>
+
+<link of the pulsar-all image>
+
 Please download the source package, and follow the README to build
 and run the Pulsar standalone service.
 ```
 
+> **NOTE**
+>
+> See [here](#9-publish-docker-images) for how to build the docker images. The release manager could push the images to a private account for others to validate.
+
 The vote should be open for at least 72 hours (3 days). Votes from Pulsar PMC members
 will be considered binding, while anyone else is encouraged to verify the release and
 vote as well.
 
 If the release is approved here, we can then proceed to next step.
 
-## 9. Promote the release
+## 7. Move master branch to next version
+
```

Comment:
We can add note : this is only for major release only.

Reply:
No. It's also required for release branches. I found it's duplicated with `20. Move branch to next version`. I will remove this part.

Reply:
I see your comment in step 20. I will update it.